data/src/main/java/hu/bsstudio/robonaut/entity/ScoreEntity.java

```diff
@@ -0,0 +1,16 @@
+package hu.bsstudio.robonaut.entity;
+
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class ScoreEntity {
+    // speed
+    private int speedScore;
+    private int speedBonusScore;
+    private List<Integer> speedTimes;
+
+    // calculated
+    private int score;
+    private int rank;
```

Comment:
@mboldi do you need this, or you will calculate it on your end? it's ok for me to store redundant information

Reply:
although i would love not to